src/commcare_cloud/ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -13,7 +13,7 @@ node.name: "{{ elasticsearch_node_name }}"
 {% if elasticsearch_version == '2.4.6'%}
 node.zone: "{{ elasticsearch_node_zone }}"
 {% endif %}
-{% if elasticsearch_version is version('7.0.0', '>=') %}
+{% if elasticsearch_version is version('5.0.0', '>=') %}
```

Comment:
I think it works fine. I found [this](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_tests.html#comparing-versions) in ansible documentation.

Reply:
Indeed. It's annoying that Ansible would violate Python semantics that way. That documentation even references a PEP about version semantics, so it is definitely in a Python context. They must be doing it by parsing the expression because [it's not possible to override the behavior of the `is` operator in Python](https://stackoverflow.com/a/3993244/10840).